Illustrations/Accuracy_vs_Sex feature/Sex_feature_vs_Accuracy.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.ensemble import RandomForestClassifier
+binwidth=0.2
+train=pd.read_csv("/home/vishal/Desktop/train(1).csv")
+train_data=train[['Sex','Survived']]
+d={"female":0,"male":1}
+try:
+    train_data['Sex']=train_data['Sex'].apply(lambda x:d[x])
+except KeyError:
+    print("There are some NaN in the input")
+plt.hist(np.array(train_data.loc[train['Survived']==1,'Sex']),bins=[d['female'],d['female']+binwidth,d['male']-binwidth,d['male']])
```

Comment:
Add some space after the exception
and document that you are plotting the histogram.

Improves readability :)

Reply:
Sure I would.